server/src/core/user/user-service.ts

```diff
@@ -10,6 +10,7 @@ import {
   createInvalidAccessTokenError, createResourceNotFoundError,
   createUniquenessFailedError,createBeneficiaryNominationFailedError } from '../error';
 import { TransactionService, TransactionCreateArgs, Transaction, InitiateDonationArgs, SendDonationArgs } from '../payment';
+import { Validator } from '.'
```

Comment:
Import the functions directly from `./validator`. e.g. `import * as validators from './validator'`.